packages/react-devtools-shared/src/devtools/views/Profiler/CommitRanked.js

```diff
@@ -91,15 +94,31 @@ type Props = {|
 function CommitRanked({chartData, commitTree, height, width}: Props) {
   const {lineHeight} = useContext(SettingsContext);
   const {selectedFiberID, selectFiber} = useContext(ProfilerContext);
+  const {
+    highlightNativeElement,
+    clearNativeElementHighlight,
+  } = useNativeElementHighlighter();
 
   const selectedFiberIndex = useMemo(
     () => getNodeIndex(chartData, selectedFiberID),
     [chartData, selectedFiberID],
   );
 
+  // Highlight last hovered element.
+  const handleElementMouseEnter = id => {
+    highlightNativeElement(id);
+  };
+
+  // remove highlighting of element on mouse leave
+  const handleElementMouseLeave = () => {
```

Comment:
Also the same, I guess we can pull these two lines out into a hook, but I also have the feeling that it will be overkill to add a custom hook just for two lines, any suggestion?